cmd/agent/flow_run.go

```diff
@@ -83,6 +85,8 @@ depending on the nature of the reload error.
 	cmd.Flags().StringVar(&r.storagePath, "storage.path", r.storagePath, "Base directory where components can store data")
 	cmd.Flags().
 		BoolVar(&r.enableDebugEndpoints, "debug.endpoints.enabled", r.enableDebugEndpoints, "Enables /debug/ HTTP endpoints")
+	cmd.Flags().
+		BoolVar(&r.disableReporting, "disable-reporting", false, "Disable reporting of enabled components to Grafana.")
```

Comment:
nit: Get the default from `r` for consistency with other flags.

```suggestion
		BoolVar(&r.disableReporting, "disable-reporting", r.disableReporting, "Disable reporting of enabled components to Grafana.")
```